vmdb/app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -12,7 +12,7 @@ def save_inventory_multi(type, parent, hashes, deletes, find_key, child_keys = [
     # Delete the items no longer found
     unless deletes.blank?
       $log.info("MIQ(#{self.name}.save_#{type}_inventory) Deleting #{self.log_format_deletes(deletes)}")
-      parent.send(type).delete(deletes)
+      parent.send(type).delete(deletes - new_records)
```

Comment:
Blah.  Thanks for reviewing this.  Now that I'm thinking about it again, I don't think this is the right fix.  I think `deletes` might be a collection proxy on Rails 4, so it gets reloaded every time `blank?` is called.

I'll verify and get back.